x-pack/plugins/fleet/server/services/epm/packages/get.test.ts

```diff
@@ -0,0 +1,171 @@
+            policy_id: '11111-111111-11111-11111',
+            enabled: true,
+            output_id: '',
+            inputs: [],
+            package: { name: 'system', title: 'System', version: '0.10.4' },
+            revision: 1,
+            created_at: '2020-12-22T21:28:05.380Z',
+            created_by: 'elastic',
+            updated_at: '2020-12-22T21:28:05.380Z',
+            updated_by: 'elastic',
+          },
+          references: [],
+          migrationVersion: { 'ingest-package-policies': '7.11.0' },
+          updated_at: '2020-12-22T21:28:05.383Z',
+          version: 'WzE1NTAsMV0=',
+          score: 0,
+        },
+        {
+          type: 'ingest-package-policies',
+          id: 'dcf83172-c38e-4501-b236-9f479da8a7d6',
+          attributes: {
+            name: 'system-3',
+            description: '',
+            namespace: 'default',
+            policy_id: '22222-22222-2222-2222',
+            enabled: true,
+            output_id: '',
+            inputs: [],
+            package: { name: 'system', title: 'System', version: '0.10.4' },
+            revision: 1,
+            created_at: '2020-12-22T21:28:05.380Z',
+            created_by: 'elastic',
+            updated_at: '2020-12-22T21:28:05.380Z',
+            updated_by: 'elastic',
+          },
+          references: [],
+          migrationVersion: { 'ingest-package-policies': '7.11.0' },
+          updated_at: '2020-12-22T21:28:05.383Z',
+          version: 'WzE1NTAsMV0=',
+          score: 0,
+        },
+        {
+          type: 'ingest-package-policies',
+          id: 'dcf83172-c38e-4501-b236-9f479da8a7d6',
+          attributes: {
+            name: 'system-4',
+            description: '',
+            namespace: 'default',
+            policy_id: '33333-33333-333333-333333',
+            enabled: true,
+            output_id: '',
+            inputs: [],
+            package: { name: 'system', title: 'System', version: '0.10.4' },
+            revision: 1,
+            created_at: '2020-12-22T21:28:05.380Z',
+            created_by: 'elastic',
+            updated_at: '2020-12-22T21:28:05.380Z',
+            updated_by: 'elastic',
+          },
+          references: [],
+          migrationVersion: { 'ingest-package-policies': '7.11.0' },
+          updated_at: '2020-12-22T21:28:05.383Z',
+          version: 'WzE1NTAsMV0=',
+          score: 0,
+        },
+      ];
+      soClient.find.mockImplementation(async ({ page = 1, perPage = 20 }) => {
+        let savedObjectsResponse: typeof savedObjects;
+
+        switch (page) {
+          case 1:
+            savedObjectsResponse = [savedObjects[1]];
+            break;
+          case 2:
+            savedObjectsResponse = savedObjects.slice(1);
+            break;
+          default:
+            savedObjectsResponse = [];
+        }
+
+        return {
+          page,
+          per_page: perPage,
+          total: 1500,
+          saved_objects: page === 3 ? [] : savedObjectsResponse,
```

Comment:
looks like page 3 will already net `[]` because of the default clause in the switch

Reply:
True. Good catch. This seems to be left over from before I implemented the `switch()` above. Removing it. :)